tao/tao_bridge/kernels/tao_compilation_cache.cc

```diff
@@ -333,6 +351,11 @@ TaoCompilationCache::TaoCompilationCache(bool async_compilation)
   graphdef_node_size_min_ = dumper_opts->graphdef_node_size_min;
   graphdef_node_size_max_ = dumper_opts->graphdef_node_size_max;
 
+  if (opts->disc_debug_mode) {
```

Comment:
Can we specify a fixed iteration number with `disc_debug_mode` ?

Reply:
Personally, I think the meaning of 'fixed iteration number' is not well defined here. The compilation cache will trigger the compilation process when the rank of the cluster is changed. In most cases, we only need to compile once.